manual/opt.tex

```diff
@@ -153,9 +153,108 @@ \subsubsection{quartic}
 \end{lstlisting}
 
 \subsubsection{adaptive}
-%To activate this optimizer, add ``-D BUILD\_LMYENGINE\_INTERFACE=1'' in the CMake command line.
 
-Manual is needed here.
+The default setting of the adaptive optimizer is to construct the linear method Hamiltonian and overlap matrices explicitly and add different shifts to the Hamiltonian matrix 
+as ``stabilizers''. The generalized eigenvalue problem is solved for each shifted Hamiltonian and updates to wave function parameters are obtained. Then a correlated sampling 
+is performed (using the middle shift's updated wave function as the guiding function) for each shift's updated wave function and the initial trial wave function.
+The cost function for these wave functions is compared, and the update corresponding to the best cost function is selected.
```

Comment:
I think the comparison of the cost function for the proposed updates can be implicitly assumed.
"The update with the best cost function is selected"